core/src/main/java/hudson/model/Cause.java

```diff
@@ -410,6 +412,11 @@ public String getUserName() {
             return userId == null ? "anonymous" : User.get(userId).getDisplayName();
         }
 
+        @Restricted(DoNotUse.class) // for Jelly
+        public String getUserUrl() {
+            return userId == null ? null : User.get(userId).getUrl();
```

Comment:
Should be http://javadoc.jenkins.io/hudson/model/User.html#get-java.lang.String-boolean-java.util.Map- instead. We do not want to occasionally re-create user instances in the case of race conditions (e.g. accessing page with `UserIdCause` in parallel with user deletion) 🐛

Reply:
`getUserName` (5 lines above) does it already, so this PR doesn't make it worse.